scripts/ChampionsRoom.asm

```diff
@@ -239,19 +239,19 @@ GaryScript_760c8:
 	ld [wJoyIgnore], a
 	ret
 
-ChampionsRoom_TextPointers:
-	dw GaryText1
-	dw GaryText2
-	dw GaryText3
-	dw GaryText4
-	dw GaryText5
+	def_text_pointers ChampionsRoom
+	dw_const ChampionsRoomGaryIntroAndAfterBattleTextScript, TEXT_CHAMPIONSROOM_GARY_INTRO_AND_AFTER_BATTLE
```

Comment:
No need for "TextScript", just "Text" or "Script". (Whichever @dannye recommended previously.)

Reply:
I used `textscript` in cases where the pointer is pointing to a label that contains a `text_asm` and `TextScriptEnd`. The main reason for this is to differentiate in cases like this: (Where you have a script and calls a text, which then farcalls).

```
BikeShopMiddleAgedWomanTextScript:
	text_asm
	ld hl, BikeShopMiddleAgedWomanText
	call PrintText
	jp TextScriptEnd

BikeShopMiddleAgedWomanText:
	text_far _BikeShopMiddleAgedWomanText
	text_end
```

Reply:
Dannye's original comments was when I attempted to rename the ScriptPointer and TextPointer tables altogether.

Reply:
If the script did something more complex than just print some text, you'd call it `BikeShopMiddleAgedWomanScript`, right? If so, I think that's fine here too, even though it's a trivial script that just prints `BikeShopMiddleAgedWomanText`.